.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "python.terminal.activateEnvInCurrentTerminal": true,
+    "python.defaultInterpreterPath": "${workspaceFolder}/venv/bin/python",
```

Comment:
I think this will cause issues on Window because the venv is located at /venv/Scripts/python.exe. I think you can specify different settings for mac/windows/linux.